go.mod

```diff
@@ -5,6 +5,7 @@ go 1.15
 require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.1
 	github.com/inconshreveable/mousetrap v1.0.0
-	github.com/spf13/pflag v1.0.5
-	gopkg.in/yaml.v2 v2.4.0
+	github.com/spf13/afero v1.8.2
+	github.com/spf13/pflag v1.0.3
```

Comment:
pflag and yaml are downgraded here...

Reply:
How did I missed that bad conflict resolution... It's fixed, thanks umarcor.